deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -87,7 +87,10 @@ spec:
                 name: windows-machine-config-operator
             spec:
               containers:
-              - command:
+              - args:
+                - --zap-level=debug
```

Comment:
Won't it increase log level whenever we're deploying operator? Perhaps we need to think of a better way of exposing these via flags so that we can enable them only we need it.

Reply:
Yes, see https://issues.redhat.com/browse/WINC-416

Reply:
@ravisantoshgudimetla I asked for this to be added until we figure out how to do that. Please see https://github.com/openshift/windows-machine-config-operator/pull/61#discussion_r433956453

Reply:
Ohh I missed it. Thanks.